.gitignore

```diff
@@ -0,0 +1,4 @@
+.idea/
+venv/
+**db.sqlite3
+*.pyc
```

Comment:
no blank line at the end of file

Reply:
![image](https://github.com/mate-academy/py-taxi-service-startproject/assets/84961728/685ebcc8-e017-4901-ade2-a2dcd37145a9)